src/renderer/helpers/api/local.js

```diff
@@ -508,8 +509,10 @@ export function parseLocalTextRuns(runs, emojiSize = 16) {
             break
           case 'WEB_PAGE_TYPE_CHANNEL': {
             const trimmedText = text.trim()
-            if (CHANNEL_HANDLE_REGEX.test(trimmedText)) {
-              parsedRuns.push(`<a href="https://www.youtube.com/channel/${endpoint.payload.browseId}">${trimmedText}</a>`)
+            // In comments, mention can be `@Channel Name` (not handle, but name)
+            if (CHANNEL_HANDLE_REGEX.test(trimmedText) || (options.looseChannelNameDetection && /^@/.test(trimmedText))) {
+              // Extra space at the end due to YT tend to include one space afterward into "channel run" `text` for "mentions"
+              parsedRuns.push(`<a href="https://www.youtube.com/channel/${endpoint.payload.browseId}">${trimmedText}</a> `)
```

Comment:
Please only add the extra space for the comments mentions, for channel handles in other places like the description, the space is usually in the other text segments, so this creates double spaces